README.md

```diff
@@ -0,0 +1,46 @@
+phpSQLiteAdmin
+==============
+
+#How to use
+phpSQLiteAdmin makes use of a simple user system. The default information is listed below:
+Default username: root
+Default password: root
+
+#Installing
+Just copy the directory phpsqliteadmin2 to your webserver. You need the PHP SQLite extension. Since PHP 5.0 this extension was bundled with PHP. Beginning with PHP 5.4, this extension is available only via PECL:
+```
+apt-get install php-pear php-dev
+pecl install sqlite
+```
+If this doesn't work, download and install manually with
+```
+cd /tmp/
+wget https://pecl.php.net/get/SQLite-1.0.3.tgz
+tar xvzf SQLite-1.0.3.tgz
+cd SQLite-1.0.3/
+phpize
```

Comment:
I cannot install php_sqlite on Ubuntu 15.10. I get an error at `make` :
```
/tmp/SQLite-1.0.3/sqlite.c: In function 'zif_sqlite_open':
/usr/include/php5/main/php_globals.h:32:29: error: 'struct _php_core_globals' has no member named 'safe_mode'
```
And many more, seems like there is no running PEAR module any more